src/pages/consumers/ConsumerCredentials.vue

```diff
@@ -28,13 +28,14 @@ import { useRouter } from 'vue-router'
 import { config } from 'config'
 import { pluginMeta } from '@/pages/plugins/PluginMeta'
 import { useI18n } from '@/composables/useI18n'
-import { apiService } from '@/services/apiService'
+import { useInfoStore } from '@/stores/info'
 import CredentialList from './CredentialList.vue'
 
 const router = useRouter()
 const { t } = useI18n()
+const infoStore = useInfoStore()
 
-const enabledPlugins = ref<string[]>([])
+const enabledPlugins = computed(() => infoStore.info?.plugins?.enabled_in_cluster ?? [])
```

Comment:
Would it be better to wrap these plugin lists as getters in the store whose values derive from the `info` state?

Reply:
Updated.